js/stdlib/ts/ALGO.ts

```diff
@@ -2296,7 +2296,7 @@ const connectAccount = async (networkAccount: NetworkAccount): Promise<Account>
 };
 
 const tokensAccepted = async (addr_: Address): Promise<Array<Token>> => {
-  const addr = protect(T_Address, addr_) as Address;
+  const addr = addressFromHex(protect(T_Address, addr_) as Address);
```

Comment:
The real function you want is `extractAddr`. Also, I'm noticing that there are a lot of functions that you've worked on lately that have the type `Account` when they are supposed to have the type `Address`... the `balanceOf` and `balancesOf` and `minimumBalanceOf`. The second argument of `transfer` should be an `Address`

Reply:
And `cbr2algo_addr` --- https://github.com/reach-sh/reach-lang/blob/master/js/stdlib/ts/ALGO.ts#L420-L421

Reply:
Give me a list of fns using Account that should use Address and I'll make another PR.

Reply:
K, we should make a Jira task for it to evaluate them. Basically, anything that READS an account rather than WRITES should be `Address` rather than `Account`